public/scripts/countryData.js

```diff
@@ -0,0 +1,90 @@
+(function(module) {
+  var origin = {};
+  origin.all = [];
+  origin.winner = '';
+  origin.diff = 1000000;
+  var guatemala = {};
+  var nicaragua = {};
+  var honduras = {};
+  guatemala.all = [];
+  nicaragua.all = [];
+  honduras.all = [];
+  guatemala.gs = [];
+  nicaragua.gs = [];
+  honduras.gs = [];
+  guatemala.diff = 0;
+  nicaragua.diff = 0;
+  honduras.diff = 0;
+
+  origin.getData = function() {
+
+    $.ajax(
+      {
+        url: '/api/origin1/',
+        type: 'GET',
+        success: function(data) {
+          origin.all = data;
+          console.log(origin.all);
+        },
+
+        error: function() {
+          console.error();
+        },
+      })
+    .done(function(n) {
+      origin.loadAll();
+      origin.compareAll(guatemala, nicaragua, honduras);
+    }
+  );
+  };
+
+  origin.loadAll = function() {
+
```

Comment:
This is pretty repetitive. You could pass in some argument when `origin.loadAll()` is called, and use that argument to run the `.filter()`. It would ultimately reduce your code by ~15 lines here.